payments-core/src/main/java/com/stripe/android/payments/core/authentication/WebIntentAuthenticator.kt

```diff
@@ -91,6 +91,13 @@ internal class WebIntentAuthenticator @Inject constructor(
                 returnUrl = null
                 shouldCancelIntentOnUserNavigation = false
             }
+            is StripeIntent.NextActionData.DisplayKonbiniDetails -> {
+                // nextActionData.hostedVoucherUrl will never be null as AuthenticatorRegistry won't direct it here
+                authUrl = nextActionData.hostedVoucherUrl.takeIf { it!!.isNotEmpty() }
+                    ?: throw IllegalArgumentException("null hostedVoucherUrl for DisplayKonbiniDetails")
+                returnUrl = null
```

Comment:
Providing a return URL would cause this to be opened in a Chrome Custom Tab instead of a web view. Should we do that?

Reply:
This is what the other vouchers do, so I just kept with it.

Reply:
We should probably start using CCT.

Reply:
Are you peaceful if I leave it like this for this PR and submit a new PR to update all voucher based LPMs to switch to CCT?

Reply:
Or rather, what's the motivation?

Reply:
I believe for one-click auth to be enabled, we need CCT.

Reply:
Also, I think some payment methods are using CCT while others are using webview right? For example, bancontact uses CCT.

Reply:
These wouldn't have auth though, you'd need to take them to a different app, or to a physical location, so they wouldn't benefit from that.